README.md

```diff
@@ -6,6 +6,12 @@ This repo is intended for tracking issues with Managed OpenShift on Azure (OSA).
 
 - Updates about the service, including new features and new Azure regions: [OpenShift feed in Azure Updates](https://azure.microsoft.com/updates/?product=kubernetes-service)
 
+## Contents
+
+- [Quickstart: Deploy a Managed OpenShift cluster](/docs/openshift-on-azure-walkthrough.md)
+- [Known Issues](/docs/known-issues.md)
+- [AAD Application Configuration](/docs/aad-application-configuration.md)
```

Comment:
Do we want to "Highlight" this doc? It could be confusing for user who just wants to Quickstart ?
We are referencing to it in the `openshift-on-azure-walkthrough.md`
My 2 Cents

Reply:
agreed, removed